apps/src/StudioApp.js

```diff
@@ -2660,6 +2662,31 @@ StudioApp.prototype.enableBreakpoints = function() {
       } else {
         this.editor.setBreakpoint(e.line);
       }
+
+      // Log breakpoints usage to firehose. This is part of the work to add
```

Comment:
do we care that this event will be logged every time the gutter is clicked? it might be interesting to differentiate between setting/clearing a breakpoint, or we might only care to log when a breakpoint is set

Reply:
ooh, I'll add that! We're just interested in general engagement with this feature for our specific purposes, but nothin' wrong with more data.